packages/stark-ui/src/modules/action-bar/components/action-bar.component.html

```diff
@@ -5,35 +5,29 @@
 	[id]="actionBarId"
 >
 	<div class="action-bar-wrapper">
-		<div
-			*ngFor="let action of actionBarConfig.actions; trackBy: trackAction"
-			[id]="actionBarId + '-' + action.id"
-			(click)="onClick(action, $event)"
-			class="stark-action-bar--action"
-		>
-			<ng-container *ngIf="action.isVisible !== false">
-				<button
-					[color]="action.buttonColor ? action.buttonColor : buttonColor"
-					[ngClass]="action.className"
-					[matTooltip]="action.labelSwitchFunction ? (action.labelActivated | translate) : (action.label | translate)"
-					[disabled]="!action.isEnabled"
-					mat-icon-button
-					type="button"
-				>
-					<mat-icon
-						[svgIcon]="action.iconSwitchFunction ? action.iconActivated : action.icon"
-						class="stark-small-icon"
-					></mat-icon>
-				</button>
-				<label
+		<ng-container *ngFor="let action of actionBarConfig.actions; trackBy: trackAction">
+			<button
+				[id]="actionBarId + '-' + action.id"
```

Comment:
Why this change? Now the one with `id` is the `<button>` and before it was the `<div>`... is this something intended? Is this tested?

Reply:
Actually, the previous implementation was wrong for 2 reasons. The `(click)` should be on the `<button>` element and not on the `<div>`. Also, when an action is **not** visible, it results like this in the previous implementation (element still exists and the click event also):
![image](https://user-images.githubusercontent.com/10333565/74929032-9702c480-53d2-11ea-88a6-60688e8ecdaf.png)

In the new one, the element is not generated at all:
```html
<!--bindings={
  "ng-reflect-ng-if": "false"
}-->
```

I tested manually the layout and the usability and everything seems ok :blush: